templates/taxi/car_detail.html

```diff
@@ -19,4 +20,16 @@
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  {% if request.user in car.drivers.all %}
+    <form method="post" action="{% url 'taxi:remove-driver' car.id %}">
+      {% csrf_token %}
+        <button type="submit" class="btn btn-danger">Видалити мене з цього автомобіля</button>
+    </form>
+  {% else %}
+    <form method="post" action="{% url 'taxi:assign-driver' car.id %}">
```

Comment:
The URL in the form action should end with a slash to follow the checklist guideline: 'Don't forget to add / at the end of the URLs.'